### DISCUSSION TITLE
It still shows the differences putting the ignore coordinates. It just yellows it.

### DISCUSSION THREAD

**Discussion by smyaseen** [red6/pdfcompare#101] [Q&A] [answered]
Hi, as the title described, the excluded part is just yellowed but the comparison is still done.  Is there any way I can totally ignore the comparisons?

See the attached picture.

![image](https://user-images.githubusercontent.com/37702376/122679699-51901a80-d205-11eb-82ba-cb797337cb20.png)

Thank you.

**Comment by finsterwalder** [comment, marked as answer]
No. Differences in ignored areas are still shown, but not counted as different. So two documents that only differ in ignored areas are reported as equal. 
You can even determine that there were differences in ignored areas by calling the method: CompareResult.hasDifferenceInExclusion()

**Reply by smyaseen** [reply]
Thank you for answering. One more question, is it possible to output only the parts which differ and not the similarities?

**Reply by finsterwalder** [reply]
Currently you can only change the color for actual and expected pixels (default red and green).
The similarities are slightly reduced in intensity so they turn to gray, but they are still visible. The amount of dimming can't be configured at the moment.
Do you really only want to see the differences? I think it's much easier to locate and understand the differences, when I can still see the whole document.

**Reply by smyaseen** [reply]
It's ok, thank you for the replies, helped me a lot!
